pkgs/development/compilers/graalvm/community-edition.nix

```diff
@@ -17,16 +17,16 @@
 , cups
 # runtime dependencies for GTK+ Look and Feel
 , enableGTK ? true
-, glib
 , cairo
+, glib
 , gtk3
 }:
 
 let
   runtimeDependencies = [
     cups
   ] ++ stdenv.lib.optionals enableGTK [
-    glib cairo gtk3
+    cairo glib gtk3
```

Comment:
looks like you forgot a `rebase --autosquash` for the fixup here.

Reply:
Squashed.